README.md

```diff
@@ -127,6 +136,33 @@ This module has a variable and an output.  This text here will be output before
 
 ```
 
+To output markdown docs as content (_Note: underscores are escaped in markdown output_):
+
+```bash
+$ terraform-docs md --markdown-content _example
+This module has a variable and an output.  This text here will be output before any inputs or outputs!
+
```

Comment:
I agree but:

- the extra empty line is inside a markdown fence, so it doesn't matter for linting
- this line was added in ccf85a95df9c55b42ba2e2f6f3e8af8b254f2b2a and it has been like this for quite a while (and I just duplicated the `table` example`
- and actually it's inconsistent with the actual `./_example/main.tf` content

I would suggest we fix this in a separate PR, because potentially we need to fix all the examples in `README` based on 1) content of `./_example/main.tf` or 2) provided [Example](https://github.com/segmentio/terraform-docs/blob/master/README.md#example) in `README`

Reply:
Agreed. I wasn't aware that this a) was already in there and b) the linter doesn't care. Thanks!